Gemfile

```diff
@@ -38,8 +38,8 @@ gem 'momentjs-rails', '>= 2.9.0' # needed for human-friendly textual dates
 gem 'font-awesome-rails'
 
 # needed to run graders in the background
-gem 'delayed_job', github: 'dsander/delayed_job', branch: 'rails51' # FIXME: github
```

Comment:
I'm pretty sure BN can't send email at the moment, actually.  I agree we shouldn't have two methods of delaying jobs, but I'm leery of switching this over right before the semester.

Question: is switching to backburner orthogonal to using Steno as the sandboxing mechanism?  If so, then that should be a separate PR so I can disentangle them in my head...